src/cozmo/world.py

```diff
@@ -102,6 +116,10 @@ def __init__(self, conn, robot, **kw):
 
         self.light_cubes = {}
 
+        #: :class:`cozmo.objects.Charger`: Cozmo's charger.
```

Comment:
there is no `cozmo.objects.Charger` class... presumably mean `ObservableObject `